Goal: Correct headers in SWAT status page

Fixes: #511275

Status wrt upstream: Will be fixed in 3.3.0

Note: 

Index: samba-3.3.0rc2/source/web/statuspage.c
===================================================================
--- samba-3.3.0rc2.orig/source/web/statuspage.c
+++ samba-3.3.0rc2/source/web/statuspage.c
@@ -427,7 +427,8 @@
 
 	printf("<h3>%s</h3>\n", _("Open Files"));
 	printf("<table border=1>\n");
-	printf("<tr><th>%s</th><th>%s</th><th>%s</th><th>%s</th><th>%s</th><th>%s</th></tr>\n", _("PID"), _("Sharing"), _("R/W"), _("Oplock"), _("File"), _("Date"));
+	printf("<tr><th>%s</th><th>%s</th><th>%s</th><th>%s</th><th>%s</th><th>%s</th><th>%s</th></tr>\n",
+		_("PID"), _("UID"), _("Sharing"), _("R/W"), _("Oplock"), _("File"), _("Date"));
 
 	locking_init_readonly();
 	share_mode_forall(print_share_mode, NULL);
